### PR TITLE
[WIP] INS-437: func tests

### PR DESCRIPTION
**Pull request by megge-dream**
closes INS-437

**- What I did**
Add some func tests for negative cases

**- How to verify it**
Run tests

### CODE REVIEW

File: functest/main_test.go [hunk 1]
```diff
@@ -352,3 +360,112 @@ func TestTransferMoney(t *testing.T) {
 	assert.Equal(t, uint(889), secondBalanceResponse.Amount)
 	assert.Equal(t, "RUB", secondBalanceResponse.Currency)
 }
+
+func TestWrongHOST(t *testing.T) {
```
Comment (LinnikD) [inline functest/main_test.go]: Useless test, lets remove it

File: functest/main_test.go [hunk 2]
```diff
@@ -90,7 +92,7 @@ func buildGinsiderCLI() error {
 
 func waitForLaunch(stdout io.ReadCloser) error {
 	done := make(chan bool, 1)
-	timeout := 30 * time.Second
+	timeout := 70 * time.Second
```
Comment (ivanshibitov) [inline functest/main_test.go]: It's very dirty hack, let's dig into problem and fix the reason

File: functest/main_test.go [hunk 3]
```diff
@@ -352,3 +360,102 @@ func TestTransferMoney(t *testing.T) {
...
+	})
+
+	response := &baseResponse{}
+	unmarshalResponseWithError(t, body, response)
+
+	assert.Equal(t, api.BadRequest, response.Err.Code)
+	assert.Equal(t, "Bad request", response.Err.Event)
+}
+
+// TODO: uncomment test after doing errors in smart contracts
```
Comment (ivanshibitov) [inline functest/main_test.go]: May be to add test with wrong Post body? I mean, incorrect json

File: testutils/core.go [hunk 4]
```diff
@@ -28,3 +29,9 @@ func RandomRef() core.RecordRef {
 	rand.Read(ref[:]) // nolint
 	return ref
 }
+
+// RandomString generates random uuid and return it as string
+func RandomString() string {
+	newUUID, _ := uuid.NewV4()
```
Comment (golangcibot) [inline testutils/core.go]: G104: Errors unhandled.
